kclhand_control/src/kclhand_controller_new.cpp

```diff
@@ -0,0 +1,1213 @@
+#include <sstream>
```

Comment:
please rename to kclhand_controller.cpp in case it is needed you can make a copy under src/backup/ but at some point we should delete the folder

Reply:
done! will clean the backup folder soon!